filesystem/src/main/java/com/dropbox/android/external/fs3/AllPersister.kt

```diff
@@ -12,7 +12,7 @@ interface AllPersister<Raw, Key> : Persister<Raw, Key>, DiskAllRead<Raw>, DiskAl
      * throw an exception
      */
     @Throws(FileNotFoundException::class)
-    override suspend fun CoroutineScope.readAll(path: String): ReceiveChannel<Raw>
```

Comment:
I don't see why this needs to be a suspend function and it caused `Ambiguous coroutineContext due to CoroutineScope receiver of suspend function`